### PR TITLE
Enable RWD exception logging with stack trace

### PR DESCRIPTION
**Pull request by kdeloach**
Connects #12

### CODE REVIEW

File: src/api/main.py [hunk 1]
```diff
@@ -67,10 +69,12 @@ def run_rwd(lat, lon):
                 shutil.rmtree(output_path)
                 return jsonify(**output)
         except:
+            log.exception('Could not GeoJSON from output')
             return error_response('Could not get GeoJSON from output.')
     except Exception as exc:
+        log.exception('Error running Point_Watershed_Function')
```
Comment (lewfish) [inline src/api/main.py]: I thought you were going to include the stacktrace?

Comment (kdeloach) [inline src/api/main.py]: It's included in the output (see: https://docs.python.org/2/library/logging.html#logging.Logger.exception).

**Comment by kdeloach** [comment]
Here is the output when executing `curl http://localhost:5000/rwd/1/2` on the host:

```
> ./scripts/run.sh 
Run the RWD Docker container.
+ docker run -v /opt/rwd-data:/opt/rwd-data -v /var/projects/rapid-watershed-delineation:/opt/rwd -p 5000:5000 --rm docker-rwd
[2016-01-15 17:59:03 +0000] [1] [INFO] Starting gunicorn 19.4.5
[2016-01-15 17:59:03 +0000] [1] [INFO] Listening at: http://0.0.0.0:5000 (1)
[2016-01-15 17:59:03 +0000] [1] [INFO] Using worker: sync
[2016-01-15 17:59:03 +0000] [9] [INFO] Booting worker with pid: 9
[2016-01-15 17:59:03 +0000] [10] [INFO] Booting worker with pid: 10
[2016-01-15 17:59:03 +0000] [19] [INFO] Booting worker with pid: 19
[2016-01-15 17:59:03 +0000] [24] [INFO] Booting worker with pid: 24
Error running Point_Watershed_Function
Traceback (most recent call last):
  File "/opt/rwd/src/api/main.py", line 60, in run_rwd
    output_path
  File "/opt/rwd/rwd/Rapid_Watershed_Delineation.py", line 78, in Point_Watershed_Function
    raise Exception('Point located outside the watershed.')
Exception: Point located outside the watershed.
```

**Comment by lewfish** [comment]
+1, tested

**Pull request merged**